snuba/web/db_query.py

```diff
@@ -59,6 +59,7 @@ def update_query_metadata_and_stats(
 def raw_query(
     request: Request,
     query: ClickhouseQuery,
+    reader: Reader,
```

Comment:
Doesn't this need a bound type variable?

Reply:
it does.